lib/rollbar/exception_reporter.rb

```diff
@@ -27,7 +27,8 @@ def report_exception_to_rollbar(env, exception)
     end
 
     def capture_uncaught?
-      Rollbar.configuration.capture_uncaught != false
+      Rollbar.configuration.capture_uncaught != false &&
+        !Rollbar.configuration.enable_rails_error_subscriber
```

Comment:
If we are subscribed to rails' error subscriber, our `capture_uncaught` setting becomes redundant?

Reply:
This prevents double sends, because the Rails error reporter (not our subscriber) re-raises in some cases.